### ISSUE TITLE
Freeze When Use CTRL Shortcuts

### ISSUE THREAD

**Issue by Tahinli** [elevenhsoft/WebApps#11]
Hi, When I use Ctrl+(x,c,v) program freezes.

https://github.com/elevenhsoft/WebApps/assets/96421894/e79c6baf-32a6-4cc7-8150-f98c761152d1

**Comment by elevenhsoft** [comment]
I know about this. Also heard it's an upstream bug. I must think what to do about it.
I need to dig it deeper. For now I did not have time, probably it needs to look into libcosmic and how it handles clipboards. It must crash on something related to it. Thank you for your testing. That's really helps and I'm gonna fix them all. For now I would recommend to avoid using your clipboard if possible. 

Unfortunately I realized it's just a one of more important features since most of the time people will copy URL instead of typing it. Well. I'm gonna to make a priority for this one.
